jupyterhub_traefik_proxy/toml.py

```diff
@@ -183,6 +186,10 @@ def get_target_data(d, to_find):
             self.routes_cache["backends"][backend_alias] = {
                 "servers": {"server1": {"url": target, "weight": 1}}
             }
+            self.log.info(
+                f"Using TOML configuration file {self.toml_dynamic_config_file} to store the routes."
+                " Make sure it's the same configuration file used by Traefik"
```

Comment:
For #74 in particular, I was thinking of including the toml dynamic path in the error logs when the wait for outputs to be added fails, rather than at info-level before attempting each time.

The case I'm thinking of is what I did the other day, where I had the wrong value here and the result was a timeout waiting for the route to show up in traefik. So I'd put it below, at `await self._wait_for_route` and if there's an error, log `"Is traefik configured to watch {self.toml_dynamic_config_file}?"`

Reply:
Updated. Thanks @minrk.